src/react-vite/03-intro-to-jsx/intro-to-jsx.md

```diff
@@ -55,62 +68,216 @@ Take the below HTML and convert it to JSX:
 
 ### Setup
 
-TODO
+We've created an assets package with images and CSS you'll need to build the application. Install the `place-my-order-assets` package.
 
 ```shell
 npm install place-my-order-assets
 ```
 
-Remove contents of `src/App.css` file:
+Remove contents of `src/App.css` file and replace with the following:
```

Comment:
For consistency right now, I would follow this format for pointing out the files for updating:

https://github.com/bitovi/academy/blob/63c1798f4982759c0c8b238da08475e546a1a72e/src/angular/3-creating-components/creating-components.md#L125

Reply:
Oh, also, we want to update `App.css` and `index.css` (see the diff code below).